package.json

```diff
@@ -127,4 +127,4 @@
     "trailingComma": "es5",
     "printWidth": 120
   }
-}
+}
```

Comment:
Nit: trailing newline

Reply:
fixed

Reply:
un-fixed. Turns out this newline is coming from the presubmit formatter, so we should just let it be.